### PR TITLE
7 - created the published schedule table

### LINKED ISSUE #7: Create Table for Published Schedule
## Instructions

- Using the DB model provided, write a `CREATE TABLE` statement in SQL for the Published Schedule table.
- Drop the table if it already exists.

## Notes

- For now, assume that all attributes should be `NOT NULL` unless otherwise specified.
- Arrows on the DB model indicate foreign key constraints
- For any `ENUM` fields, make sure to define them in a line above the `CREATE TABLE` statement.

## Acceptance Criteria

- SQL creation code should exist in a file located in `server/schema/published_schedule.sql`

## Resources
- [DB Model](https://www.figma.com/file/NDPdo1MR13qgsB4LubtzyZ/Database-Model?type=whiteboard&node-id=0-1&t=m1J0vcLI2mzIb5SN-0)
- [Create Table Tutorial](https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-create-table/)
- [Creating Enums Tutorial](https://www.postgresql.org/docs/current/datatype-enum.html)

### PR DESCRIPTION
**Pull request by arralia**
closes issue #7

### CODE REVIEW

File: server/schema/published_schedule.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
+DROP TABLE [IF EXISTS] published_schedule;
```
Comment (ThatMegamind) [inline server/schema/published_schedule.sql]: The brackets around [IF EXISTS] are mainly for readability in tutorials, but they're not actually part of the syntax. Try removing the brackets to remove the eslint issues!

File: server/schema/published_schedule.sql [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+CREATE TYPE year AS ENUM ('Junior','Senior');
+
+CREATE TABLE [IF NOT EXISTS] published_schedule (
```
Comment (michellelin1) [inline server/schema/published_schedule.sql]: the brackets aren't needed here. the brackets in the tutorial indicate optional fields but when you actually use those options, you don't use the brackets

**Review by ThatMegamind — CHANGES_REQUESTED** [review]
Looks pretty good, just some minor comments. I think Michelle will also leave some feedback

**Review by ThatMegamind — APPROVED** [review]
Looks good to me!

**Pull request merged**